scripts/properties2json.js

```diff
@@ -13,12 +12,20 @@ var supportedLocales = env.get('SUPPORTED_LOCALES') || "*";
 function getDirectoryLocales(src) {
   return FS.listDirectoryTree(path.join(process.cwd(), src)).then(function(dirTree) {
     var list = [];
-    dirTree.forEach(function(i) {
-      var that = i.split(src + '/');
```

Comment:
This was the actual problem - rather than splitting on a known string `src`, we were splitting on a string that was not guaranteed to resolve. The fix was to split on only the known string, and then process the result such that it conforms to the locale code format.